src/Certificate.php

```diff
@@ -70,9 +62,13 @@ private function load()
     public function isExpired()
     {
         $now = new \DateTime('now');
-        return $this->validFrom <= $now && $this->validTo >= $now;
+        return $this->validTo < $now;
```

Comment:
tem inicio e fim certo, mas não e logico que se a data final do Certificado for menor que  HOJE, ele não está expirado? 2010 < 2016 = true; Que diferença faz se o certificado foi criado em 2006? Ou estou completamente enganado!